modules/widget-visibility/widget-conditions.php

```diff
@@ -286,7 +293,11 @@ public static function widget_conditions_admin( $widget, $return, $instance ) {
 							</div>
 
 							<div class="condition-control">
-								<span class="condition-conjunction"><?php echo esc_html_x( 'or', 'Shown between widget visibility conditions.', 'jetpack' ); ?></span>
+								<span class="condition-conjunction">
+									<?php
+									//	echo esc_html_x( 'or', 'Shown between widget visibility conditions.', 'jetpack' );
```

Comment:
Ideally, we would use a bit of Javascript to alternate this value between "or" and "and" depending on the state of the checkbox that was introduced.

If we are going to omit this label instead, we should remove this line rather than comment out the text.